client/lib/plans/features-list.js

```diff
@@ -1021,7 +1021,10 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_PLAN_SECURITY_DAILY ]: {
 		getSlug: () => constants.FEATURE_PLAN_SECURITY_DAILY,
 		getIcon: () => 'lock',
-		getTitle: () => i18n.translate( 'All Security Daily features' ),
+		getTitle: () =>
+			( {
+				spp: i18n.translate( 'All Jetpack Security features' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'All Jetpack security features for comprehensive protection' )` *ES Score: 7*